.travis/prepare_os_osx.sh

```diff
@@ -7,6 +7,10 @@ for tool in automake libtool pkg-config libffi gmp openssl node python3; do
     brew install ${tool} || brew upgrade ${tool} || true
 done
 
+# create links so python 3 tools get used
+ln -sf /usr/local/bin/pip3 $HOME/.bin/pip
```

Comment:
Lol. So what happens? It tries to use python2 pip? Or has no pip at all?

Reply:
yeah, by default macOS is python 2.7. And the homebrew version only installs `python3` and `pip3`, so we need to create links for `python` and `pip`.